src/main/java/org/skriptlang/skript/util/Builder.java

```diff
@@ -23,7 +23,7 @@ interface Buildable<B extends Builder<B, T>, T> {
 		 * @return A builder representing this object.
 		 */
 		@Contract("-> new")
-		B builder();
```

Comment:
what's the reason for this change

Reply:
it's a bit too simplistic as a name (and some of these classes often have static methods like `builder`, e.g. `ClassLoader.builder()`). I think `toBuilder` reflects the method's purpose well (if not better)